cypress/e2e/settings/tenant/tenant-view.cy.js

```diff
@@ -38,6 +38,8 @@ describe('Permissions -> Tenant', () => {
     { tags: [TestTypes.extendedPath, DevTeams.firebird] },
     () => {
       cy.intercept('/location-units/institutions*', { locinsts: [testData.institution] });
+      // reload is needed because sometimes Location setup section is not displayed
+      cy.reload();
```

Comment:
Let's use  cy.visit(SettingsMenu.tenantLocationsPath); instead of cy.reload(); and remove navigation from before hook